src/udev/udevadm-control.c

```diff
@@ -37,6 +46,8 @@ static bool arg_exit = false;
 static int arg_max_children = -1;
 static int arg_log_level = -1;
 static int arg_start_exec_queue = -1;
+static int arg_has_control_commands = false;
```

Comment:
Hmm, does this need to be a dedicated variable? Just check for each option instead?